packages/mobile/src/recipients/RecipientPicker.tsx

```diff
@@ -47,37 +46,29 @@ interface Props {
   isOutgoingPaymentRequest: boolean
 }
 
-interface StateProps {
-  recipientInfo: RecipientInfo
-  showSendToAddressWarning: boolean
-}
+function RecipientPicker(props: RecipientProps) {
```

Comment:
No need to change it here because it'd be annoying, just mentioning it for other situations, but the pattern we usually do it destructuring the props here, like `function RecipientPicker({ searchQuery, sections, etc }: Props) {` instead of using the `props` thing directly.

I'm not really sure if one or the other is better, just sharing how we usually do it